internal/pkg/inject/offset_results.json

```diff
@@ -7312,13 +7192,6 @@
               {
                 "field": "buckets",
                 "offsets": [
-                  {
```

Comment:
This will also fix the failure in https://github.com/open-telemetry/opentelemetry-go-instrumentation/actions/runs/9848477538/job/27190668050 (it caused us to fail read the tracers map)